plugins/test/runner.proto

```diff
@@ -147,7 +147,13 @@ message Env {
   // Log file to receive plugin output. Can be /dev/std{out,err}.
   string log_path = 5;
   // Fixed timestamp in Unix seconds. If unset it will be the Unix epoch.
-  uint64 time_secs = 6;
+  optional uint64 time_secs = 6;
+  // Number of background streams to run in benchmarks. If unset, no background
+  // streams will be run.
+  optional uint64 num_background_streams = 7;
```

Comment:
naming bikeshed: The term "background" implies multithreading to me, and happily that is not how this works. Maybe something like 'other_streams' or 'memory_pressure_streams' or 'unmeasured_streams'?

Reply:
Changed to "additional streams"